perllib/FixMyStreet/App/Controller/Admin/Reports.pm

```diff
@@ -330,6 +330,11 @@ sub edit : Path('/admin/report_edit') : Args(1) {
         } else {
             $problem->unset_extra_metadata('closed_updates');
         }
+        if ($c->get_param('send_state') && ($c->get_param('send_state') ne $problem->send_state)) {
+            $c->forward('/auth/check_csrf_token');
+            $problem->send_state($c->get_param('send_state'));
+            $c->forward( '/admin/log_edit', [ $id, 'problem', 'send_state_change'] );
```

Comment:
Do we think this deserves a special log entry? I guess mark as sent/resend do, and it's not really an "edit" is it.